string detector_name
string robot_name
---
bool ack

